test/integration/targets/canonical-pep517-self-packaging/minimum-build-constraints.txt

```diff
@@ -0,0 +1,19 @@
+# Lowest set in `pyproject.toml`:
+setuptools == 39.2.0; python_version < "3.9"
+
+# Lowest supporting Python 3.10:
+setuptools == 57.0.0; python_version == "3.9" or python_version == "3.10"
+
+# Lowest supporting Python 3.11:
+setuptools == 60.0.0; python_version >= "3.11"
```

Comment:
Where did these minimum versions come from? Do Python 3.9 and 3.10 share the same minimum version?

Reply:
This is mostly coming experimentation and checking the changelogs. Some other versions explode in weird ways (because of bugs and/or depreciations) that I got tired of trying to hack into the working state.

So with this, I was try to bisect what works under different interpreter versions. And with that `< 3.9` pin I was trying to get as close as possible to what `pyproject.toml` specifies.

Reply:
Oh, and it's not something that this PR changes — if you try pinning setuptools to v39.2.0 on devel, build will traceback too. This is not something that we smoke-test but with this PR, we will. The reason nobody reports this is that PEP 517 pull in the latest setuptools for the given platform.
I can imagine that on some downstreams it might work, maybe, when they call legacy `setup.py` directly or use older Python, so I'm not looking into bumping it in `pyproject.toml`, plus it's out of the scope here anyway. Feature-wise, the currently set value is enough, though.